resources/config_configuration_recorders.go

```diff
@@ -88,6 +89,9 @@ func fetchConfigConfigurationRecorders(ctx context.Context, meta schema.ClientMe
 
 func generateConfigRecorderArn(_ context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	cl := meta.(*client.Client)
-	cfg := resource.Item.(types.ConfigurationRecorder)
-	return resource.Set(c.Name, client.GenerateResourceARN("config", "config-recorder", *cfg.Name, cl.Region, cl.AccountID))
+	cfg, ok := resource.Item.(types.ConfigurationRecorder)
+	if ok {
```

Comment:
To make it Go-idiomatic the happy path needs to be the "unindented" one, so it should go like:

- run something
- check if error, { error handling block }
- continue processing

So this block would be:
```go
cfg, ok := resource.Item.(types.ConfigurationRecorder)
if !ok {
    return fmt.Errorf("not config config recorder")
}
return resource.Set(c.Name, client.GenerateResourceARN("config", "config-recorder", *cfg.Name, cl.Region, cl.AccountID))
```

Also in all `if ok` checks below... So, you should move error handling inside the if statement and the happy path outside.

Reply:
Thank you for clear feedback!! I have applied the change